services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/presigner/DefaultS3Presigner.java

```diff
@@ -98,9 +98,7 @@
 @SdkInternalApi
 public final class DefaultS3Presigner extends DefaultSdkPresigner implements S3Presigner {
     private static final AwsS3V4Signer DEFAULT_SIGNER = AwsS3V4Signer.create();
-    private static final S3Configuration DEFAULT_S3_CONFIGURATION = S3Configuration.builder()
-            .checksumValidationEnabled(false)
-            .build();
+    private static final S3Configuration DEFAULT_S3_CONFIGURATION = S3Configuration.builder().build();
```

Comment:
Instead of changing the default configuration, should we just validate the customized service configuration and log a warning message or something if checksum is enabled on line 117 ?